pysumma/ensemble.py

```diff
@@ -218,16 +218,26 @@ def attribute_product(list_config):
             {'attributes': d} for d in product_dict(**list_config)}
 
 
-def total_product(dec_conf={}, param_conf={}, attr_conf={}):
+def file_manager_product(list_config):
+    return {'++'+'++'.join('{}={}'.format(k, v) for k, v in d.items())+'++':
+            {'file_manager': d} for d in product_dict(**list_config)}
+
+
+def total_product(dec_conf={}, param_conf={}, attr_conf={}, fman_conf={},
+                  sequential_keys=False):
     full_conf = deepcopy(dec_conf)
     full_conf.update(param_conf)
     full_conf.update(attr_conf)
+    full_conf.update(fman_conf)
     prod_dict = product_dict(**full_conf)
     config = {}
-    for d in prod_dict:
+    for i, d in enumerate(prod_dict):
         name = '++' + '++'.join(
-            '{}={}'.format(k, v) if k in param_conf or k in attr_conf else v
+            '{}={}'.format(k, v) if k in param_conf or k in attr_conf
```

Comment:
need to add `k in dec_conf`.

Reply:
I think we don't actually need to have `k in dec_conf` here since it's somewhat redundant to store both the key and value for decisions. There are a couple of decisions options whose key could ambiguous, such as `zeroFlux` or `jrdn1991` but generally that information is stored in the attributes of the outputs so I'm less inclined to need to store it in the file names as well. Then, this option just gets put into the `else` clause below and written out as just the value.